### ISSUE TITLE
Panic on `Execute? [y/n] ` after `edit` command

### ISSUE THREAD

**Issue by hymkor** [hymkor/sqlbless#3]
```
$ sqlbless  sqlite3 mytestdb
# SQL-Bless v0.15.1-3-g90aa9e1-windows-amd64 by go1.23.1
  Ctrl-M or      Enter: Insert Linefeed
  Ctrl-J or Ctrl-Enter: Exec command

SQL> edit t
  2> ;
n             s
0             hogehogt
2             hogehoge2
1             hogehoge3
3             hogehoge4
Apply the changes ? [y/n] y

---
UPDATE  t
   SET  s = 'hogehogt'
 WHERE  n = 0
   AND  s = 'hogehogx'
Execute? [y/n] panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x4ea2f4]

goroutine 1 [running]:
github.com/mattn/go-tty.(*TTY).raw(0x0)
        C:/Users/hymkor/go/pkg/mod/github.com/mattn/go-tty@v0.0.7/tty_windows.go:380 +0x34
github.com/mattn/go-tty.(*TTY).Raw(0xc00203e000?)
        C:/Users/hymkor/go/pkg/mod/github.com/mattn/go-tty@v0.0.7/tty.go:20 +0x13
github.com/nyaosorg/go-readline-ny.GetKey({0xb5e400, 0x0})
        C:/Users/hymkor/go/pkg/mod/github.com/nyaosorg/go-readline-ny@v1.4.0/getkey.go:18 +0x43
github.com/nyaosorg/go-readline-ny.(*_Tty).GetKey(0x0?)
        C:/Users/hymkor/go/pkg/mod/github.com/nyaosorg/go-readline-ny@v1.4.0/tty8.go:40 +0x1d
github.com/hymkor/sqlbless.(*InteractiveIn).GetKey(0x0?)
        C:/Users/hymkor/src/sqlbless/loop.go:254 +0x1f
github.com/hymkor/sqlbless.ask2({0xa7cbb5?, 0xc0000560b0?}, {0xa72454, 0x2}, {0xa72442, 0x2}, 0xc000151358)
        C:/Users/hymkor/src/sqlbless/edit.go:60 +0xb6
github.com/hymkor/sqlbless.askSqlAndExecute({0xb5f6d8, 0x1d25700}, 0xc00015ce80, 0xc000151358, {0xc00203c000, 0x46})
        C:/Users/hymkor/src/sqlbless/edit.go:82 +0x118
github.com/hymkor/sqlbless.doEdit.func8(0xc0000b2150)
        C:/Users/hymkor/src/sqlbless/edit.go:306 +0xc65
github.com/hymkor/csvi.(*_Application).Each(0xc00008e160?, 0xc000151678)
        C:/Users/hymkor/go/pkg/mod/github.com/hymkor/csvi@v1.10.1/rowptr.go:100 +0x9c
github.com/hymkor/sqlbless.doEdit({0xb5f6d8, 0x1d25700}, 0xc00015ce80, {0xc00008e160, 0x7}, {0xb5e340, 0xc0000086c0}, {0xb5b360, 0xc0000560b0})
        C:/Users/hymkor/src/sqlbless/edit.go:236 +0xbf9
github.com/hymkor/sqlbless.(*Session).Loop(0xc00015ce80, {0xb5f6d8, 0x1d25700}, {0xb5e340, 0xc0000086c0}, 0x0)
        C:/Users/hymkor/src/sqlbless/loop.go:380 +0x2a16
github.com/hymkor/sqlbless.Config.Run({{0x0, 0x0}, {0xa7240d, 0x1}, 0x0, {0xa73ce0, 0x6}, 0x0, {0xb56610, 0x1}, ...}, ...)
        C:/Users/hymkor/src/sqlbless/loop.go:566 +0x717
github.com/hymkor/sqlbless.Main()
        C:/Users/hymkor/src/sqlbless/main.go:108 +0x325
main.main()
        C:/Users/hymkor/src/sqlbless/cmd/sqlbless/main.go:16 +0x13
exit status 2
```

**Comment by hymkor** [comment]
+ v0.15.1 → NG (panic)
+ v0.15.0 → NG (panic) go-tty v0.0.7, go-readline-ny v1.4.0
+ v0.14.0 → OK (not panic) go-tty v0.0.5 , go-readline-ny v1.3.1
+ v0.13.0 → OK (not panic)

**Comment by hymkor** [comment]
[Device not closed after close tty · Issue #50 · mattn/go-tty](https://github.com/mattn/go-tty/issues/50)

**Comment by hymkor** [comment]
tty オブジェクトの使いまわしが問題だった模様
